src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -42,6 +44,43 @@
 _RESPONSE_PAYLOAD_SIZE = 42
 
 
+class Test_OngoingCalls(unittest.TestCase):
```

Comment:
optional: Should we split it into another file?

Reply:
technically the `_OngoingCalls` is implemented as within the `channel_test.py`, having the feeling that we are not really following the convention of one python file one test file. If you want I can move the whole testing to a new file called `chanel_close_test.py`

Reply:
I'm in favor of splitting a new file. The `call_test.py` we had is too long for new comers to understand.